assertion/function/assertiontree/backprop_util.go

```diff
@@ -749,3 +750,22 @@ func CheckGuardOnFullTrigger(trigger annotation.FullTrigger) annotation.FullTrig
 	}
 	return trigger
 }
+
+// addAssignmentToConsumer updates the consumer with assignment entries for informative printing of errors
+func addAssignmentToConsumer(lhs, rhs ast.Expr, pass *analysis.Pass, consumer annotation.ConsumingAnnotationTrigger) (err error) {
```

Comment:
nit: we can drop the named return since it's not necessary :)

Reply:
Agreed. Done :)